cmd/autoheal/awx_job.go

```diff
@@ -162,3 +171,7 @@ func (h *Healer) checkAWXJobStatus(jobID int) (finished bool, err error) {
 
 	return
 }
+
+func (r AWXJobRunner) AWX() *config.AWXConfig {
```

Comment:
Using this method makes calls shorter, but also less clear: it is no longer explicit that you are using the configuration. I'd suggest to avoid that.